scripts/music.sh

```diff
@@ -113,6 +115,17 @@ main() {
   local app_name="$(printf "%s" "$music_data" | awk 'NR==6')"
   clock 'extract data'
 
+
+  # Check if track duration is 0 (state in-between a "play/pause" and "no-player")
+  if test "$track_duration" -eq 0; then
```

Comment:
suggestion(optional): for this one, I would preferred to have another state, 'stopped'.

See [AppleScript player](https://github.com/spywhere/tmux-now-playing/blob/master/players/applescript.js#L57) implementation here, it does produce a stopped state. However, this might not be for all players.

If you are not comfortable to update it, feel free to skip this one.